Formula/arx-libertatis.rb

```diff
@@ -25,6 +25,23 @@ class ArxLibertatis < Formula
   def install
     args = std_cmake_args
 
+    # The patches for these aren't straightforward to backport because of
+    # other changes; these minimal inreplaces get it building.
+    # HEAD is fine, and the next stable release will contain these changes.
```

Comment:
Mind creating "Can haz tag plz?" issue so we can at least track where they stand on this?